libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -552,17 +575,27 @@ if(LIBC_TYPES_HAS_FLOAT16)
     libc.src.math.canonicalizef16
     libc.src.math.ceilf16
     libc.src.math.copysignf16
+    # TODO: aarch64 bug
+    # Please see https://github.com/llvm/llvm-project/pull/100632#issuecomment-2258772681
+    # libc.src.math.expf16
     libc.src.math.f16add
     libc.src.math.f16addf
+    # libc.src.math.f16addl
     libc.src.math.f16div
     libc.src.math.f16divf
+    # libc.src.math.f16divl
+    # libc.src.math.f16fma
```

Comment:
I think you can enable `f16fma`. We lost the Buildbot logs, but we still have a GitHub comment from me saying that the issue was ISel failing with `fp_round` from `f128` to `f16`: https://github.com/llvm/llvm-project/pull/97072#discussion_r1658925698.

I added `f16fma` in the same PR as `f16fmal` and `f16fmaf128`, that's why I disabled all three of them. Then I realized that `double` shouldn't cause issues, so I only enabled `float16` narrowing functions that take `float` or `double` on AArch64 in following PRs, and it didn't cause issues.